nemo_text_processing/inverse_text_normalization/de/data/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
```

Comment:
it will fail if i do that, since the test is not updated yet. Should be done best in separated PR since once test is adjusted all headers need to be changed

Reply:
updated